src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -1,15 +1,169 @@
 import { ArrowForwardIcon } from '@chakra-ui/icons'
-import { Button, HStack, Stack } from '@chakra-ui/react'
+import {
+  Box,
+  Button,
+  Flex,
+  HStack,
+  Skeleton,
+  SkeletonCircle,
+  Tag,
+  TagLabel
+} from '@chakra-ui/react'
+import { AprTag } from 'plugins/cosmos/components/AprTag/AprTag'
+import { StakingAction } from 'plugins/cosmos/components/modals/Staking/Staking'
+import { useMemo } from 'react'
 import { NavLink } from 'react-router-dom'
+import { Column } from 'react-table'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
 import { Card } from 'components/Card/Card'
-import { Text } from 'components/Text'
+import { ReactTable } from 'components/ReactTable/ReactTable'
+import { RawText, Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
 
-import { StakingOpportunitiesRow } from './StakingOpportunitiesRow'
+type StakingOpportunity = {
+  id: number
+  validatorName: string
+  apr: number
+  stakedAmount?: string | undefined
+  stakedRewards?: string | undefined
+  fiatRewards?: number | undefined
```

Comment:
IMO this should all come to the component as `BigNumber`s

Reply:
Also, we probably shouldn't pass the `fiatRewards` directly - it should be done by  calculating `cryptoAmount.times(fiatRate)`. You can grep for `fiatRate` to see how this is done across the codebase.